_search-plugins/search-relevance/index.md

```diff
@@ -98,4 +98,62 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
 {: .note}
 
 You cannot save a given comparison for future use, so Compare Search Results is not suitable for systematic testing.
-{: .note}
\ No newline at end of file
+{: .note}
+
+## Comparing search results with the OpenSearch Intelligent Ranking plugin
+
+The OpenSearch Intelligent Ranking plugin takes search results returned from OpenSearch and processes them using the Amazon Kendra intelligent enterprise search service. The Amazon Kendra service leverages semantic search capabilities to re-rank OpenSearch results, considering the context of the OpenSearch query and relevant documents from OpenSearch.
+
+To use the OpenSearch Intelligent Ranking plugin, you must first set up the Amazon Kendra service. To get started, see [Amazon Kendra](https://aws.amazon.com/kendra/). For plugin setup instructions, see [OpenSearch Intelligent Ranking plugin](https://docs.aws.amazon.com/kendra/latest/dg/opensearch-rerank.html).
+
+Once you've set up the OpenSearch Intelligent Ranking plugin, you can compare search results from OpenSearch and Amazon Kendra by entering an OpenSearch query in **Query 1** and an Intelligent Ranking query in **Query 2**.
+
+### Example
+
+The following example searches for the text "data" in the `oci` index. 
```

Comment:
Are we keeping 'oci' as the index for sure?

Reply:
Changed to suggested example. @msfroh: would it be possible for you to get the same image but resize the window so it's narrower? Otherwise, the text is unreadable because our body pane is very narrow on the doc site. Thanks!